src/index.html

```diff
@@ -22,6 +26,412 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="container_logo"
+      >
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+      <nav class="container_menu">
+        <ul class="menu">
+          <li>
+            <a
+              class="is-active"
+              href="#"
+            >
+              APPLE
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__imac">
+          <img
+            class="card__imac-img"
+            src="../src/images/imac.jpeg"
+            alt="imac_photo"
+          />
+        </div>
+        <div class="card__info">
+          <h1 class="card__title">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </h1>
+          <h2 class="card__text-code">Product code: 195434</h2>
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <h2 class="card__reviews">Reviews: 5</h2>
+          </div>
+          <div class="card__price">
+            <h2 class="card__price-name">Price:</h2>
+            <h1 class="card__price-number">$2,199</h1>
+          </div>
+          <div class="card__button">
+            <a
+              class="card__button-link"
+              href="#"
+              data-qa="card-hover"
```

Comment:
The 'data-qa' attribute is used for testing purposes and should not be present in the production code unless it's being specifically used for test automation.